mas-foundation/src/main/java/com/ca/mas/core/io/ssl/TLSSocketFactory.java

```diff
@@ -21,7 +23,9 @@
     private static final String SSL_TLS_V1_PROTOCOL = "TLSv1";
     private static final String SSL_TLS_V1_1_PROTOCOL = "TLSv1.1";
     private static final String SSL_TLS_V1_2_PROTOCOL = "TLSv1.2";
+    private static final String SSL_TLS_V1_3_PROTOCOL = "TLSv1.3";
     private static final String[] SUPPORTED_TLS =  {SSL_V3_PROTOCOL, SSL_TLS_V1_PROTOCOL, SSL_TLS_V1_1_PROTOCOL, SSL_TLS_V1_2_PROTOCOL};
+    private static final String[] SUPPORTED_TLS_Q =  {SSL_V3_PROTOCOL, SSL_TLS_V1_PROTOCOL, SSL_TLS_V1_1_PROTOCOL, SSL_TLS_V1_2_PROTOCOL, SSL_TLS_V1_3_PROTOCOL};
```

Comment:
SUPPORTED_TLS_Q , I think you are trying to say, its android q onwards, but its difficult to understand still, please use a more meaningful keyword here.

Reply:
Can I make SUPPORTED_TLS_FROM_ANDROID_Q ?